src/hooks/useMagicAutofocus.ts

```diff
@@ -100,7 +100,7 @@ export default function useMagicAutofocus(
           timeout = setTimeout(() => {
             triggerFocus();
             delay = false;
-          }, 200);
+          }, 400);
```

Comment:
This works because we changed the animation in the navigation upgrade and it is taking a bit longer than previously. It is a seriously volatile piece of code and preferably we should change it going in the future. Especially going with the new navigators.